src/Sylius/Component/Core/Model/ShippingMethod.php

```diff
@@ -45,4 +46,12 @@ public function setZone(ZoneInterface $zone)
 
         return $this;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getTranslationEntityClass()
+    {
+        return parent::getTranslationEntityClass();
```

Comment:
Because the translation class for ShippingMethod is not in the Core/Model, but in the Shipping/Model